nextflow.config

```diff
@@ -323,7 +323,7 @@ manifest {
 
 // Nextflow plugins
 plugins {
-    id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-schema@2.2.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
as of today, the downgrade is not needed anymore. You can add the latest version back if you want :)

Reply:
Thanks!